src/SFML/Main/MainWin32.cpp

```diff
@@ -41,6 +41,11 @@
 
 #include <windows.h>
 
+// Inform the Nvidia driver that this SFML application should
+// make use of the more powerful discrete GPU
+extern "C" {
+__declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
```

Comment:
Remind me which one they were? I can't remember any of them regarding C linkage specifiers. :grin: